packages/lesswrong/components/common/TabNavigationMenu/menuTabs.jsx

```diff
@@ -0,0 +1,276 @@
+import React from 'react';
+import { Components } from 'meteor/vulcan:core';
```

Comment:
Is there a reason this is menuTabs.jsx instead of MenuTabs.jsx?

Reply:
Cause it's not a component proper. It has jsx in it, but otherwise it's like withVote.js or similar.